### PR TITLE
Feature/update conf

### PR DESCRIPTION
**Pull request by j8xixo12**
I have added the projects and training pages. Currently, the training promotion material is not ready, so I have marked it as Coming Soon until the promotion material is available.

### CODE REVIEW

**Review by rockleona — APPROVED** [review]
The preview LGTM, I think will wait others approve then merge.

File: contents/home.mdx [hunk 1]
```diff
@@ -1,5 +1,25 @@
 # science, code, and open source
 
+## Sciwork Conference 2024
+Welcome to the **Sciwork Conference 2024**! Join us for an exciting day of training, and collaboration opportunities with experts and enthusiasts from the tech community.
+
+## NASA TOPS Training
+
+Please refere <a href="/training">Training</a>
```
Comment (YenPeiChen07) [inline contents/home.mdx]: There is a typo need to be fixed with the change `Please refer to the <a href="/training">Training</a> page.`

**Review by YenPeiChen07 — CHANGES_REQUESTED** [review]
Thanks @j8xixo12, it looks good. There is only one typo need to fix as the comment below.

Additionally, I found that the event date is wrong and the Discord link is invalid in our homepage.
image

Could you help update this, or do you know someone who could assist?

**Comment by j8xixo12** [comment]
> Thanks @j8xixo12, it looks good. There is only one typo need to fix as the comment below.
> 
> Additionally, I found that the event date is wrong and the Discord link is invalid in our homepage. image
> 
> Could you help update this, or do you know someone who could assist?

I have corrected the event in https://github.com/sciwork/swconf2024/pull/8/commits/a913f0ea09b5aba3e0f9842fadfef0b727af0f9b , and this commit was contained in #8.

File: contents/home.mdx (hunk 1 above)
Comment (j8xixo12) [reply contents/home.mdx]: Thanks for pointing out this typo, I have corrected these typo in latest commit.

**Review by jack482653 — APPROVED** [review]

**Pull request merged**